packages/dynamic-import/server.js

```diff
@@ -66,7 +66,11 @@ function middleware(request, response) {
   response.setHeader("Access-Control-Allow-Origin", "*");
 
   if (request.method === "OPTIONS") {
-    response.setHeader("Access-Control-Allow-Headers", "*");
+    if (request.headers["access-control-request-headers"] !== undefined) {
+      response.setHeader("Access-Control-Allow-Headers", request.headers["access-control-request-headers"]);
```

Comment:
Especially if we end up having to check for multiple variations of this header, it would be good to store the value in a variable that we could reuse down here.